src/controllers/HealthController.js

```diff
@@ -19,7 +19,7 @@ async function checkHealth (req, res) {
   checksRun += 1
   const timestampMS = new Date().getTime()
   try {
-    await service.searchMembers(null, { page: 1, perPage: 1 })
+    await service.getMember(null, "Ghostar", { })
```

Comment:
What does the service return if the member handle isn't found?

Reply:
Health check will fail with a 503 and a message like:

```
{
    "message": "There is database operation error, Member with handle: \"Ghostarasdfasdf\" doesn't exist"
}
```